latex/slides/07_arrays.tex

```diff
@@ -227,20 +227,21 @@ \subsection{}
 \begin{frame}[fragile]{Remember, Remember}
 	Old and busted:
 	\begin{itemize}
-		\item The size of variables may differ on different architectures
-		\item The size of a char is \textbf{always} 1 byte
+		\item Number of bytes per type may differ on different architectures
+		\item A char is 1 byte long (but can be longer in some cases)
```

Comment:
I misunderstood the definitions. A `byte` can change in size, but a `char` is bound to the size of a `byte`. I fixed the original commit via an interactive rebase.